src/Main.java

```diff
@@ -1,22 +1,34 @@
+import java.util.ArrayList;
+import java.util.List;
+
 public class Main {
     public static void main(String[] args) {
-        Sorting sort = new Sorting();
+        SortPlayers sortPlayers = new SortPlayers();
+        List<Player> players = new ArrayList<>();
 
-        sort.add(new Player("Robert", 100));
-        sort.add(new Player("Johny", 234));
-        sort.add(new Player("MutherFucker666", 512));
-        sort.add(new Player("EasyKill", 101));
-        sort.add(new Player("PussyMagness", 230));
+        players.add(new Player("Robert", 100));
+        players.add(new Player("Johny", 234));
+        players.add(new Player("BigStan", 512));
+        players.add(new Player("Easyy", 101));
+        players.add(new Player("King", 230));
+
+        System.out.println("Lista nieposortowana: ");
+        players.forEach(System.out::println);
 
-        sort.getPlayers().forEach(System.out::println);
-        System.out.println();
-        sort.sortByScoreDown();
         System.out.println();
-        sort.sortByScoreUp();
+        sortPlayers.sortByScoreDescending(players);
+        players.forEach(System.out::println);
```

Comment:
You are printing players over and over so you can move it the new method instead of calling println all the time :)

Reply:
I made static method in Main class